src/examples/liquid_fill_gauge/liquid_fill_gauge.ts

```diff
@@ -0,0 +1,261 @@
+// Global values provided via the API
+declare var looker: Looker
+declare var LookerCharts: LookerChartUtils
+declare var require: any
+declare var _: any
+
+import * as d3 from 'd3'
+import { handleErrors } from '../common/utils'
+/**
+ * TODO install this version?
+ * https://github.com/ugomeda/d3-liquid-fill-gauge
+ * hmm, it's not published on npm
+ */
```

Comment:
TODO here.

You can still add the repo as a dependency in package.json: https://stackoverflow.com/a/38731705

Alternatively this seems to be present in https://photino.github.io/d3-components/api/liquid-fill-gauge.html